src/app/components/chat/chat.component.html

```diff
@@ -1 +1,47 @@
-<p>chat works!</p>
+<div class="h-svh w-svw bg-slate-100 flex flex-col">
+  @if (messages$ | async; as messages) {
+  <div
+    id="messages"
+    class="flex flex-col p-3 w-svw min-h-3.5 my-14 overflow-y-scroll overflow-x-hidden"
+  >
+    @for (message of messages; track $index) { @if (message.userId) {
```

Comment:
Don't ever track by index please. Same as react. It's a pointless tracker and should be anything else. Track by the whole message object even.